DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -132,7 +132,9 @@
     <![CDATA[superClusterRelinked_.reset();]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="14">
+  <class name="pat::Jet"  ClassVersion="16">
+   <version ClassVersion="16" checksum="727883729"/>
+   <version ClassVersion="15" checksum="3059757499"/>
```

Comment:
Where is version 15 coming from since the version number is incremented from 14 to 16? Is this for backward compatibility with previous releases?